jetty-core/jetty-http2/jetty-http2-tests/pom.xml

```diff
@@ -23,6 +24,7 @@
           <reportsDirectory>${project.build.directory}/h2spec-reports</reportsDirectory>
           <excludeSpecs>
             <excludeSpec>3.5 - Sends invalid connection preface</excludeSpec>
+            <excludeSpec>8.1.2.3 - 8.1.2.6. Malformed Requests and Responses</excludeSpec>
```

Comment:
Well done to find the magic needed!     
But can we have a TODO to investigate and remove the exclusions... and/or an issue.

Reply:
We can use issue #8212, right?

Reply:
@gregw *bump*

Reply:
Ok, don't close it with this